dev/build.featureStart.base/src/com/ibm/ws/test/featurestart/features/FeatureFilter.java

```diff
@@ -67,9 +67,8 @@ public static String skipFeature(String shortName) {
             // location variable.
             return "Required variable 'wmqJmsClient.rar.location' is not set";
 
-        } else if (shortName.equalsIgnoreCase("mpHealth") ||
-                   shortName.equalsIgnoreCase("mpMetrics")){
-            // Versionless feature cannot run by itself
+        } else if (shortName.indexOf("-") == -1 ) {
```

Comment:
We should add a helperMethod isVersionless(String) to make it obvious what this is doing...